### ISSUE TITLE
If multiple rpms are produced they are all deployed with the same artifactId

### ISSUE THREAD

**Issue by ProggerPete** [ctron/rpm-builder#69]
I'm producing a number of rpms with different package names.  However the artifact uploaded to the repo always has the name of the module.

How do I specify the artifactId for the different rpms I produce?

**Comment by ctron** [comment]
yes, the package name has nothing to do with the maven artifact ID. To my knowledge, there can only be one combination of groupId+artifactId+version+classifier+type. Normally that is something like: `foo/bar/baz/1.0/baz-javadoc.jar`

But I think you can use something like the `build-helper` to attach additional artifacts: https://www.mojohaus.org/build-helper-maven-plugin/attach-artifact-mojo.html

**Comment by ProggerPete** [comment]
Fair enough, I was hoping for some sort of <artifactId> config option to make my life easy.  I'll just give them all different classifiers instead.

**Comment by ctron** [comment]
I guess it could be solved in the RPM builder plugin too. PRs welcome :grin:

Not sure it that works in your case, but you also can set the classifier: https://ctron.github.io/rpm-builder/rpm-mojo.html#classifier
